pkg/operator/ceph/object/rgw.go

```diff
@@ -315,6 +317,22 @@ func BuildDomainName(name, namespace string) string {
 	return fmt.Sprintf("%s-%s.%s.%s", AppName, name, namespace, svcDNSSuffix)
 }
 
+// ParseDomainName parse the name and namespace from the dns name
+func ParseDomainName(domainName string) (types.NamespacedName, error) {
```

Comment:
Do we have unit tests for this?

Reply:
yes. here: https://github.com/rook/rook/blob/5d5905c5f1da46911ed3d94897e10a32c5f7734f/pkg/operator/ceph/object/notification/controller_test.go#L453